shared/src/ledger/mod.rs

```diff
@@ -6,4 +6,6 @@ pub mod native_vp;
 pub mod parameters;
 pub mod pos;
 pub mod storage;
+#[cfg(any(feature = "ibc-vp", feature = "ibc-vp-abci"))]
+pub mod token;
```

Comment:
should we also prefix this?
```suggestion
pub mod ibc_token;
```

Reply:
Thank you for your suggestion! It looks clearer.
I've moved the token native VP under `ibc::vp`.